website/landingpage/public/index.html

```diff
@@ -32,6 +32,9 @@
               class="close-btn hide-for-xlarge"
             ></button>
             <ul class="menu">
+              <li class="item">
+                <a href="/about" class="">About Luigi</a>
```

Comment:
This link, at least locally running gulp, is not valid:
![Bildschirmfoto 2019-10-23 um 09 50 33](https://user-images.githubusercontent.com/11227688/67370246-9e2bda00-f57a-11e9-9f1c-ea0b3b1eb1e2.png)

Reply:
Thank you for review, the bug will be fixed within the follow-up ticket https://github.com/SAP/luigi/issues/921